documentation/docs/user_guide/obsolete_and_deprecated_features/deprecated_features.md

```diff
@@ -0,0 +1,37 @@
+# Deprecated features for CABLE users #
```

Comment:
Same comment about merging the pages for deprecated and obsolete features.

Reply:
I have merged deprecated and obsolete together.